src/experiments/tagging/6_27_17/testrotation_0.json

```diff
@@ -0,0 +1,19 @@
+{
+    "batch_size": 8,
+    "git_commit": "95f4e9",
```

Comment:
I'm not sure what the intention is behind these experiment files, especially why you need four of them to test whether this works. I think you should explain this in the commit message.